google/cloud/storage/examples/storage_grpc_samples.cc

```diff
@@ -72,10 +70,27 @@ void GrpcClientWithProject(std::string project_id) {
   std::cout << "Successfully created a gcs::Client configured to use gRPC\n";
 }
 //! [grpc-client-with-project]
+
 #else
+
+void GrpcReadWrite(std::string const&) {}
+void GrpcClientWithDP() {}
 void GrpcClientWithProject(std::string const&) {}
+
 #endif  // GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
 
+void GrpcReadWriteCommand(std::vector<std::string> argv) {
+  if (argv.size() != 1 || argv[0] == "--help") {
+    throw examples::Usage("grpc-read-write <bucket-name>");
+  }
+  GrpcReadWrite(argv[0]);
+}
+
+void GrpcClientWithDPCommand(std::vector<std::string> const& argv) {
+  if (argv.empty()) return GrpcClientWithDP();
+  throw examples::Usage("grpc-client-with-dp");
```

Comment:
Nit: Change the order to match the others so the exception is not the flow-through case?

Reply:
Fixed.